src/main/java/org/trigon/hardware/phoenix6/Phoenix6Inputs.java

```diff
@@ -30,15 +35,29 @@ public void toLog(LogTable table) {
         latestTable = table;
     }
 
+    /**
+     * Registers a threaded signal.
+     * Threaded signals use threading to process certain signals separately, keeping them running smoothly in the background.
+     *
+     * @param statusSignal         the signal
+     * @param updateFrequencyHertz the update frequency in hertz
+     */
```

Comment:
add @ override, no need for duped jdoc

Reply:
override what? There isn't a registerSignal in BaseInputs

Reply:
Mb, keep the javadoc then. 
Make sure everything is document properly and consistently.